gitpod-io/website
# Add instructions for how to use Cognito for SSO

## Description

This PR adds AWS Cognito as an additional SSO provider option that can be used with Gitpod Dedicated. This is especially, useful in situations where a customer/prospect does not want to fully integrate into their IDP yet, and just wants something that works. 
<img width="644" alt="Screenshot 2023-09-08 at 15 56 15" src="https://github.com/gitpod-io/website/assets/22516873/8c78770b-a2f6-40aa-a32b-5b8374146d05">


### How to test: 
1. Open netlify preview, navigate to docs>dedicated>guides>getting started guide - or go here: https://deploy-preview-3967--gitpod-kumquat.netlify.app/docs/gitpod-dedicated/guides/getting-started#3-set-up-gitpod
2. Then navigate down to the `3. Set Up Gitpod step`
3. Open configure Single Sign-On and open the AWS Cognito toggle
4. Review guide, ensure the content is correct, and language, clarity and grammar are on point



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3967"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
